### ISSUE TITLE
In variable pricing table remove repeatable field should be a button, not a link

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4343]
In general links are for links when you actually go somewhere, buttons are for actions. Here is the [reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/#semantics-for-controls).

![remove_repetable_field](https://cloud.githubusercontent.com/assets/1820415/13252918/d73b3e9c-da41-11e5-88a0-8055ac6a12c5.png)

Code for remove "button" looks like this at the moment.

```
<td>
	<a href="#" class="edd_remove_repeatable" data-type="price" style="background: url(<?php echo admin_url('/images/xit.gif'); ?>) no-repeat;">&times;</a>
</td>
```

Screen readers see that as "times" and it's not clear what that actually do. I suggest that we change it to button like this:

```
<td>
	<button class="edd_remove_repeatable" data-type="price" style="background: url(<?php echo admin_url('/images/xit.gif'); ?>) no-repeat;"><span class="screen-reader-text"><?php printf( __( 'Remove price option %s', 'easy-digital-downloads' ), $key ); ?></span><span aria-hidden="true">&times;</span></button>
</td>
```

This would need a small CSS change/addition like this:

```
.edd_remove_repeatable {
    border: none;
    padding: 0;
}
```

**Comment by samikeijonen** [comment]
This is not related but `edd-admin.css` could be coded using [CSS standard](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/) :)

**Comment by samikeijonen** [comment]
The same should be done in Download Files metabox for remove "buttons".

**Comment by pippinsplugins** [comment]
:+1:

**Comment by pippinsplugins** [comment]
PR works great, merged into `release/2.6`.
